synapse/lib/cell.py

```diff
@@ -98,8 +99,8 @@ class CellApi(s_base.Base):
 
                 async def makeWidget(wvalu, wtype):
                     # This will throw if the user doesn't have the appropriate widget permission
-                    await self._reqUserAllowed('widget', wtype)
-                    return await self.cell.makeWidget(wvalu, wtype)
+                    await self._reqUserAllowed(('widget', wtype))
```

Comment:
We should fix the funcdef above to include self

Reply:
ok